.github/update-layers.yaml

```diff
@@ -0,0 +1,27 @@
+name: Update Lambda Layers
+# This workflow runs early on the 3rd of every month.
+# It runs the copy-layer.sh bash script for prod and nonprod
+
+on:
+  schedule:
+    - cron:  '11 2 3 * *'
```

Comment:
For everyone's sanity, add a comment here with the human-readable interpretation of this.

Reply:
lol, k